src/utils.ts

```diff
@@ -290,3 +296,86 @@ export const findLastIndex = <T>(
   }
   return -1;
 };
+
+// Export functions
+
+type ExportOpts = {
```

Comment:
this is file where we keep our common utilities so we might not want to keep on adding more and more utils to this file. We can move this to `export.ts` as it will make more sense there ? 
We will anyways end up exporting utils from different files which we want to add in `excalidraw-utils`.
what do you think ?
cc @dwelle

Reply:
I'd actually do the opposite, keep it in this file, and also move it outside `src/`. What we want is a clear signal these functions shouldn't be used internally.

Reply:
By moving them outside `src`, they won't even show in intellisense which is what we want.

Reply:
> By moving them outside src, they won't even show in intellisense which is what we want.
You mean only moving these new utils outside src right ?

Reply:
yes